### PR TITLE
Mokkery Gradle Plugin

### PR DESCRIPTION
**Pull request by kigya**
For the Docosaurus website, added a description of what needs to be considered when working with Mokkery and Convention Plugins.

image

### CODE REVIEW

**Review by lupuuss — CHANGES_REQUESTED** [review]
Thanks you for contribution! I've recently discovered this inconvenience. For know, I will add this section, but I will try to remove this inconvenience in the future.

File: website/docs/Setup.mdx [hunk 1]
```diff
@@ -59,6 +59,29 @@ pluginManagement {
...
+        }
+    }
+}
+```
+
+
+Once the required Maven repository is connected, you will be able to successfully download the Mokkery plugin dependency.
+```kotlin
+dependencies {
+    implementation("dev.mokkery:mokkery-gradle:2.1.1)
```
Comment (lupuuss) [inline website/docs/Setup.mdx:81]:
```suggestion
    implementation("dev.mokkery:mokkery-gradle:2.1.1")
```

File: website/docs/Setup.mdx [hunk 2]
```diff
@@ -59,6 +59,29 @@ pluginManagement {
...
 ```
+
+For those using convention plugins, make sure that the dependencies configuration includes the path to the required Maven repository.
+
+```kotlin
+dependencyResolutionManagement {
+    repositories {
+        mavenCentral()
+        // required to download gradle plugin
+        maven {
```
[the remark this answers is not in the document]
Comment (kigya) [reply website/docs/Setup.mdx]: Yes, I have tested it and it works. Good point

File: website/docs/Setup.mdx [hunk 3]
```diff
@@ -59,6 +59,29 @@ pluginManagement {
     }
 }
 ```
+
```
[the remark this answers is not in the document]
Comment (kigya) [reply website/docs/Setup.mdx:62]: Extracted as a separate section

File: website/docs/Setup.mdx (hunk 1 above)
Comment (kigya) [reply website/docs/Setup.mdx]: Good catch, fixed

**Comment by kigya** [comment]
Updated version:

**Review by lupuuss — APPROVED** [review]

**Pull request merged**